### PR TITLE
Update and add readme.nodejs.md files with latest generator arguments

### PR DESCRIPTION
**Pull request by ghost**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/datalake-analytics/data-plane/readme.nodejs.md [hunk 1]
```diff
@@ -7,9 +7,22 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
...
   batch:
-    - output-folder: $(node-sdks-folder)/lib/services/dataLake.Analytics/lib/catalog
-      tag: package-catalog
-    - output-folder: $(node-sdks-folder)/lib/services/dataLake.Analytics/lib/job
-      tag: package-job
+    - tag: package-catalog
+    - tag: package-job
 ```
+
+``` yaml $(nodejs) && $(tag) == 'package-catalog'
```
Comment (RikkiGibson) [inline specification/datalake-analytics/data-plane/readme.nodejs.md]: Beyond the output-folder changing to source-code-folder-path, is this a style change or a correctness change to add the additional yaml blocks?

**Review by RikkiGibson — APPROVED** [review]

File: specification/datalake-analytics/data-plane/readme.nodejs.md (hunk 1 above)
Comment (daschult) [reply specification/datalake-analytics/data-plane/readme.nodejs.md]: @lmazuel showed me to do it this way when I configured the Java Swagger To SDK readme files and I feel like it's a little easier to read (style change). I'm not sure if one way is more correct than the other.

File: specification/cognitiveservices/data-plane/EntitySearch/readme.nodejs.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   package-name: azure-cognitiveservices-entitysearch
-  output-folder: $(node-sdks-folder)/lib/services/entitySearch/lib
+  package-version: 1.0.0-preview
+  output-folder: $(node-sdks-folder)/lib/services/entitySearch
```
Comment (amarzavery) [inline specification/cognitiveservices/data-plane/EntitySearch/readme.nodejs.md]: @daschult - FTYI, We have rollup packages for different cognitive services category ([search](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/cognitiveServicesSearch), [vision](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/cognitiveServicesVision), [language](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/cognitiveServicesLanguage)). Somehow we will have to make sure that the rollup packages also get updated and published when we generate a package for an individual congnitive service.

**Review by amarzavery — APPROVED** [review]

File: specification/datalake-analytics/data-plane/readme.nodejs.md (hunk 1 above)
Comment (lmazuel) [reply specification/datalake-analytics/data-plane/readme.nodejs.md]: In this particular scenario, I feel both are fine.
However, for multi-api versionning, we want to have ONE "batch" at the beginning of the file to explain what ApiVersions are shipped by all SDKs. Then, this batch will be just a list of tags.

**Review by lmazuel — APPROVED** [review]
I don't get the NodeJS specific details, but in terms of configuration looks good.

**Review by schaabs — APPROVED** [review]

**Comment by daschult** [comment]
I think this looks good. I'm going to merge it.

**Pull request merged**